shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -373,15 +375,17 @@ public void process(ReplyProcessor cp) {
         public void writeFields(DataOutput out) throws IOException {
             out.writeLong(writerId.getMostSignificantBits());
             out.writeLong(writerId.getLeastSignificantBits());
+            out.writeLong(requestId);
             out.writeLong(eventNumber);
             out.writeUTF(serverStackTrace);
         }
 
         public static WireCommand readFrom(ByteBufInputStream in, int length) throws IOException {
             UUID writerId = new UUID(in.readLong(), in.readLong());
+            long requestId = in.readLong();
             long eventNumber = in.readLong();
```

Comment:
This needs to go last with an "available" check just like serverStackTrace.

Reply:
Done. Moved it to last to make backward compatible.